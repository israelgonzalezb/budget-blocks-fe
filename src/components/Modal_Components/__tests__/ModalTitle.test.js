
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ModalTitle from '../ModalTitle';


describe('ModalTitle test', () => {
  it('ModalTitle should match snapshot', () => {
    const component = renderer.create(<ModalTitle
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
