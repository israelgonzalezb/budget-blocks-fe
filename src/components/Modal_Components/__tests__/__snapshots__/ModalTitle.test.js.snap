// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalTitle test ModalTitle should match snapshot 1`] = `
<div
  className="MuiDialogTitle-root customized-dialog-title"
>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    <p
      className="MuiTypography-root customized-dialog-title MuiTypography-body1"
    />
  </h6>
</div>
`;
